### PR TITLE
Merchant user

### PR DESCRIPTION
**Pull request by GarrettCottrell**
--Add button or link to "fulfill"  item on orders show page
--Create routing for that link to go back to order show page
--Update order page and show the item is now fulfilled
--Create flash message indicating that I have fulfilled that item
--The item's inventory quantity is permanently reduced by the user's desired quantity

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
Comment (GregJMitchell) [inline .vscode/launch.json]: Add your .vscode file to .gitignore please and do not push it up

**Review by jakeheft — APPROVED** [review]
Everything looked good - worked driver/navigator with Garrett

File: app/views/merchant/orders/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<% @order.items_by_merchant(current_user.merchant_id).each do |item| %>
+  <%# make the below conditional into a method %>
```
Comment (GregJMitchell) [inline app/views/merchant/orders/show.html.erb:2]: When are you going to do this?

File: config/routes.rb [hunk 3]
```diff
@@ -41,6 +41,8 @@
   namespace :merchant do
     get "/", to: "dashboard#show"
     get "/items", to: "items#index"
+    get "/orders/:order_id", to: "orders#show"
+    patch "/orders/:order_id", to: "item_orders#update"
```
Comment (GregJMitchell) [inline config/routes.rb]: we could do this with resources to save line(s)

**Review by GregJMitchell — CHANGES_REQUESTED** [review]
Over all looks good. Nice use of query params in the URI. There was a section that had a comment and I was wondering if you wanted to leave that in or not? If it is mean for later than I am ok merging this. Also we need to get the .vscode file out of the PR

**Pull request merged**